lib/matplotlib/axes/_axes.py

```diff
@@ -677,6 +677,14 @@ def axhline(self, y=0, xmin=0, xmax=1, **kwargs):
         -------
         :class:`~matplotlib.lines.Line2D`
 
+        Other parameters
```

Comment:
The docs seem to have built, but I am surprised that numpydoc is ok with a lowercase 'p' here.